AEXML/AEXML.swift

```diff
@@ -264,7 +264,7 @@ public class AEXMLElement: Equatable {
     /// Same as `xmlString` but without `\n` and `\t` characters
     public var xmlStringCompact: String {
         let chars = NSCharacterSet(charactersInString: "\n\t")
-        return join("", xmlString.componentsSeparatedByCharactersInSet(chars))
+        return "".join(xmlString.componentsSeparatedByCharactersInSet(chars))
```

Comment:
Should it be "xmlString.componentsSeparatedByCharactersInSet(chars).joinWithSeparator("")"?